### PR TITLE
Add bogdan to team page

### PR DESCRIPTION
**Pull request by ChingChang9**
Preview:

image

Is the order okay?

### CODE REVIEW

File: src/views/Team.vue [hunk 1]
```diff
@@ -52,13 +52,11 @@ export default {
...
-          title: "Marketing Manager",
-          photo: "parshva",
-          story: ["Parshva Shah started volunteering for PlantForever during the summer of 2021 where he helped with tree transplantation. " +
-            "He became interested in developing marketing strategies to help PlantForever's services reach more people. " +
-            "He likes problem-solving and finding innovative solutions which makes the Head of Marketing the perfect position for him. " +
-            "He develops different marketing strategies to connect homeowners with PlantForever products and services."]
+          name: "Bogdan Voroniak",
+          title: "VP of Communications",
+          photo: "bogdan",
+          story: ["Bogdan Voroniak joined the team in 2022 to further integrate his interest in environment with marketing and social media. " +
```
Comment (marmikpatel19) [inline src/views/Team.vue:58]: environment -> environmentalism

File: src/views/Team.vue [hunk 2]
```diff
@@ -52,13 +52,11 @@ export default {
...
+            "He manages all team-wide and external communications in addition to overseeing both social media and marketing progress of PlantForever."]
```
Comment (marmikpatel19) [inline src/views/Team.vue:59]: He manages all team-wide and external communications, and oversees the progress of PlantForever's social media and marketing.

**Comment by marmikpatel19** [comment]
Order doesn't matter (from an organizational point of view). You can even put me at the very bottom if you'd like :p Common equality-based approaches (as opposed to hierarchical) that I've seen in other organizations are alphabetical and random. 

Thanks!

**Comment by ChingChang9** [comment]
cool, thanks!

**Pull request merged**